src/locales/en/translation.json

```diff
@@ -206,6 +206,11 @@
     },
     "backButton": "Back to Merchants",
     "giftButton": "Gift Meal",
+    "viewMoreButton": "View More",
```

Comment:
do we need these for `cn` translations too?

Reply:
I'm actually not sure and included it to be thorough.